### PR TITLE
Update to TrackV20script

### PR DESCRIPTION
**Pull request by aquaalex**
TrackV20Script.js was updated to remove spelling mistake and add material from other V20 books.

Is it possible remove the Turning Undead Script from this pull? Sorry I am new to Git and I am still busy completing the code for the Turn Undead script (AD&D 2nd)

### CODE REVIEW

**Comment by SteveKoontz** [comment]
Unfortunately not. You have to remove the changes from your fork and then submit a new pull request. Do you want me to sit on this until you've completed your changes for the AD&D script or close this one and let you try again?

**Comment by aquaalex** [comment]
Hi Steve please just close it and let me try again once I have completed TurnUndead script
